docs/src/pages/ui/primitives/menu/react.mdx

```diff
@@ -0,0 +1,224 @@
+import {
+  AlignExample,
+  BasicExample,
+  MenuExample,
+  ControlledExample,
+  SizeExample,
+} from './examples/';
+
+import { Example } from '@/components/Example';
+
+Menu provides an accessible, interactive menu for selecting actions within an application. Dropdown menu is collision-aware and will automatically change location based on available space.
+
+## Demo
+
+<MenuDemo />
+
+## Usage
+
+Import the `Menu` and `MenuItem` components and styles. Use `MenuItem` component to configure menu options.
+
+```tsx file=./examples/basicExample.tsx
+
+```
+
+<Example>
+  <BasicExample />
+</Example>
+
+### Size
+
+Control the size of the Menu buttons and items using the `size` prop. Available options are `small`, none (default), and `large`.
+
+```tsx file=./examples/sizeExample.tsx
+
+```
+
+<Example>
+  <SizeExample />
+</Example>
+
+### Align
+
+Alignment of menu with button trigger can be controlled using the `align` prop. Available options are `start` (default), `center`, and `end`.
+
+```tsx file=./examples/alignExample.tsx
+
+```
+
+<Example>
+  <AlignExample />
+</Example>
+
+### Customize trigger button
+
+The default trigger button can be overwritten using the `trigger` prop on `Menu` with the `MenuButton` component.
+`MenuButton` can take all the same props as [Button](./button).
+
+```tsx file=./examples/menuExample.tsx
+
+```
+
+<Example>
+  <MenuExample />
+</Example>
+
+### Controlled Menu
+
+Create a controlled `Menu` using the `isOpen` and `onOpenChange` props.
+
+```tsx file=./examples/controlledExample.tsx
+
+```
+
+<Example>
+  <ControlledExample />
+</Example>
+
+## CSS Styling
+
+### Target classes
+
+| Class                  | Description                                                             |
+| ---------------------- | ----------------------------------------------------------------------- |
+| `amplify-menu`         | Menu content container (`Flex`)                                         |
+| `amplify-menu__item`   | Menu item button (`MenuButton`, wrapped in `amplify-menu` class)        |
+| `amplify-menu-trigger` | Menu trigger button (`MenuButton`, not wrapped in `amplify-menu` class) |
+
+### Global styling
+
+To override styling on all Menus, you can set the Amplify CSS variables with the built-in `.amplify-menu` class.
+
+```css
+/* styles.css */
+.amplify-menu {
+  --amplify-components-button-color: var(--amplify-colors-brand-secondary-90);
+}
+.amplify-menu-trigger {
+  --amplify-components-button-border-color: var(
+    --amplify-colors-brand-secondary-90
+  );
+  --amplify-components-button-color: var(--amplify-colors-brand-secondary-90);
+}
+```
+
+```jsx
+import { Menu, MenuItem } from '@aws-amplify/ui-react';
+import '@aws-amplify/ui-react/styles.css';
+
+import './styles.css';
+
+<Menu>
+  <MenuItem>Download</MenuItem>
+  <MenuItem>Create a Copy</MenuItem>
+  <MenuItem>Mark as Draft</MenuItem>
+</Menu>;
+```
+
+<Example>
+  <Menu
+    className="global-styling-menu"
```

Comment:
You could set the className on the `<Example>` and then it would function more like the example code

Reply:
I could move `triggerClassName` but not `className`. The reason is that `className` gets added to the Menu Content, which is rendered in a React Portal outside the Example div, so I wouldn't be able to target the content from a class on the Example div.